README.md

```diff
@@ -97,137 +53,89 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **[Add new books, games or music albums, and see the ones previously added, and they are saved to JSON files.]**
+- **[Create authors, genres and labels to sort the various items, also stored in JSON.]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+None currently available.
```

Comment:
- [ ] [OPTIONAL] I understand that you have attached the link to the video presentation to the PR description, however, we could put the video presentation link here since we do not currently have the live demo. This will give visitors an insight of what the application is about and how it works.

Reply:
Thank you @Whoistolu for the review. We will update the readme file with a demo video as you have suggested